app/views/katello/api/v2/repositories/show.json.rabl

```diff
@@ -15,6 +15,11 @@ attributes :gpg_key_id
 attributes :content_id, :content_view_version_id, :library_instance_id
 attributes :product_type
 attributes :promoted? => :promoted
+attributes :distribution_version
+attributes :distribution_arch
+attributes :distribution_bootable? => :distribution_bootable
+attributes :distribution_family
+attributes :distribution_variant
```

Comment:
Do you think we should only include the distribution nodes in the API response if there is a distribution?